django/db/models/base.py

```diff
@@ -70,22 +70,22 @@ def __setstate__(self, args):
 
 class ModelBase(type):
     """Metaclass for all models."""
-    def __new__(cls, name, bases, attrs):
+    def __new__(cls, name, bases, attrs, **kwargs):
         super_new = super().__new__
 
         # Also ensure initialization is only performed for subclasses of Model
         # (excluding Model class itself).
         parents = [b for b in bases if isinstance(b, ModelBase)]
         if not parents:
-            return super_new(cls, name, bases, attrs)
+            return super_new(cls, name, bases, attrs, **kwargs)
```

Comment:
A test is missing for this change.

Reply:
Is `test_doesnt_work_with_init_subclass` meant to test this change? I still don't see any test failures if this change is reverted.

Reply:
I thought that always passing kwargs to type was more consistent but initialization of Model itself without subclassing is impossible due to Model instance not having _meta attribute. I can't think of other way to test this change. Should I remove it?

Reply:
If it doesn't provide any useful functionality, I'd omit it.

Reply:
Ok, thanks.